data/coalition missions.txt

```diff
@@ -108,6 +109,7 @@ mission "Coalition: Contributor"
 		government "Coalition"
 	destination "Ring of Friendship"
 	on offer
+		log "After doing a few dozen jobs for the Coalition, they're finally wiling to let me purchase some of their technology. Doesn't sound like they'll sell me any of their weapons, though."
```

Comment:
Yep, apologies about missing the on offers/completes in a few of these. And fair point, MZ. So want your quote to end with something like "Bloody tyrants", but maybe a little too snarky and sweary? Not sure how much silliness you want to go for, or exactly what the swearing rules are.